src/components/ViewBook.js

```diff
@@ -9,11 +9,12 @@ function ViewBook({
   id,
   author,
   title,
+  category,
   completion,
 }) {
   return (
-    <div className="bookContainer">
-      <BookDetails title={title} author={author} id={id} />
+    <div className="bookContainer animate flip">
+      <BookDetails title={title} author={author} id={id} category={category} />
       <CompletionPercentage completion={completion} />
       <Progress />
```

Comment:
### OPTIONAL

- [ ] Kindly add the border between the CompletionPercentage and Progress div, this will make your design look EXACTLY like the figma design, see screenshots below good job so far 👍🏿 💪🏿

Zephin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/172829579-f1189637-1427-417d-83f3-03990f9c3ae6.png) |  ![Screenshot_238](https://user-images.githubusercontent.com/69638013/172829626-6a3f6b44-92c9-4f42-8c50-2a1500432c64.png)